.gitignore

```diff
@@ -22,6 +22,10 @@ DerivedData
 *.xcuserstate
 project.xcworkspace
 
+# CocoaPods
+Podfile.lock
+Pods
```

Comment:
The `Podfile.lock` will cause massive conflict from different MR contributors.

Since we follows the sem version and this is used for example, ignore Podfile.lock is a better idea for avoid this. See SDWebImage's itself project maintain guideline.

Reply:
If we don't have a Podfile.lock then we can't have reproducible builds / tests. Although this project does not have testing of the native side at the moment anyways...

I'd be alright not checking it in until native testing is implemented.

Thanks for updating this, looks great!